azure-pipelines.yml

```diff
@@ -58,14 +61,14 @@ steps:
   inputs:
     targetType: filePath
     filePath: ./build.ps1
-    arguments: '-Target $(CakeTarget) -ScriptArgs ''--CkBranch="$(Branch.CkEditor)"'',''--CdfBranch="$(Branch.CdfProvider)"'''
+    arguments: '--target=$(CakeTarget) --verbosity=diagnostic --CkBranch="$(Branch.CkEditor)" --CdfBranch="$(Branch.CdfProvider)"'

```

Comment:
Do we want to default to `--verbosity=diagnostic`?   If so, add it to the first Cake call, also

Reply:
I would vote no, logging takes time and the build would be faster without it. We can always temporarily add it if we have build issues... IMO

Reply:
I moved it to a build-time variable, and defaulted to normal

Reply:
Also CKEditor is now part of this repo and we no longer pull from a branch, so that could go away. I think that variable was removed from the build scripts already so passing it actually does nothing anymore

Reply:
I don't see any usage of `CkBranch` or `CdfBranch`.  @donker do you know where we'd expect those to be used?  Are we still pulling CDF dynamically during build/packaging?